doc/source/tune/tutorials/tune-checkpoints.rst

```diff
@@ -1,17 +1,46 @@
 A Guide To Using Checkpoints
 ============================
 
+.. _tune-two-types-of-ckpt:
+
+Two different types of Tune checkpoints
+---------------------------------------
+
+There are mainly two types of checkpoints that Tune maintains: Experiment-level checkpoints and Trial-level
+checkpoints.
+Experiment-level checkpoints save the experiment state. This includes the state of the searcher/scheduler,
+the list of trials with their statuses (PENDING, RUNNING, TERMINATED, ERROR), and the
+metadata that is pertained to each trial (hyperparameter configuration, trial logdir, etc).
+
+The experiment-level checkpoint is saved by the driver.
+The frequency at which it is conducted is automatically
+adjusted so that at least 95% of the time is used for handling training results and scheduling.
+This time can also be adjusted with the
+:ref:`TUNE_GLOBAL_CHECKPOINT_S <tune-env-vars> environment variable`.
+
+The purpose of the experiment checkpoint is to maintain a global state from which the whole Ray Tune experiment
+can be resumed from if it is interrupted or failed.
+It can also be used to analysis tuning results after a Ray Tune finished.
+
+Trial-level checkpoints capture the per-trial state. They are saved by the trainable itself.
```

Comment:
Should we mention anything about syncing to the driver here?

Reply:
There is a big section "Checkpointing and synchronization" right below this section, so I think it should be fine.